packages/truffle-core/lib/debug/mocha.js

```diff
@@ -0,0 +1,163 @@
+const colors = require("colors");
+const util = require("util");
+
+const { CLIDebugger } = require("./cli");
+const execute = require("truffle-contract/lib/execute");
+
+class CLIDebugHook {
+  constructor(config, compilation, runner) {
+    this.config = config;
+    this.compilation = compilation;
+    this.runner = runner; // mocha runner (**not** lib/test/testrunner)
+  }
+
+  async debug(operation) {
+    // turn off timeouts for the current runnable
+    // HACK we don't turn it back on because it doesn't work...
+    // tests that take a long time _after_ debug break just won't timeout
+    this.disableTimeout();
+
+    const { txHash, result, method } = await this.invoke(operation);
+
+    this.printStartTestHook(method);
+
+    const interpreter = await new CLIDebugger(this.config, {
+      compilation: this.compilation
+    }).run(txHash);
+    await interpreter.start();
+
+    this.printStopTestHook(operation.method);
```

Comment:
An argument is passed here to `printStopTestHook` even though `printStopTestHook` takes no arguments?

Reply:
```suggestion
    this.printStopTestHook();
```